spec/support/doc_auth_image_fixtures.rb

```diff
@@ -31,6 +31,14 @@ def self.selfie_image_multipart
     Rack::Test::UploadedFile.new(fixture_path('selfie.jpg'), 'image/jpeg')
   end
 
+  def self.error_yaml_multipart
+    path = File.join(
+      File.dirname(__FILE__),
+      '../fixtures/ial2_test_credential_forces_error.yml',
+    )
+    Rack::Test::UploadedFile.new(path, Mime[:yaml])
```

Comment:
ooo good call on the `Mime[:yaml]`

Reply:
> ooo good call on the `Mime[:yaml]`

Yeah, I'd had some trouble finding a good sense of what the MIME type _should_ be, and even though Rails does make a choice (`application/x-yaml` and `text/yaml`), it feels like something which could change over time if there's broader standardization ([example](https://mailarchive.ietf.org/arch/msg/media-types/e9ZNC0hDXKXeFlAVRWxLCCaG9GI/)).

I'd hoped to use this in the actual form logic as well (`file.content_type == Mime[:yaml]`), but in testing, it appears that from my browser client the real YAML data is being received as `application/octet-stream`. Hence settling on testing the file name extension.

Reply:
When we were working on this initially we observed a bunch of super inconsistent behavior around the MIME types for the YAML files. We saw everything from `application/x-yaml` to `application/text`.